code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -889,7 +889,7 @@
 			return
 
 	// CONVERT TURF
-	if(!isturf(target) && !HAS_ATOM_PROPERTY(target,PROP_ATOM_FLOCK_THING) && !istype(target, /obj/structure/girder))
+	if(!isturf(target) && (!HAS_ATOM_PROPERTY(target,PROP_ATOM_FLOCK_THING) || istype(target, /obj/lattice/flock)) && !istype(target, /obj/structure/girder))
```

Comment:
`obj/lattice/flock` should have the `PROP_ATOM_FLOCK_THING`

Reply:
I'm pretty sure it does

Reply:
oh wait it's not'd. NOT has_atom OR istype